include/igl/copyleft/comiso/miq.cpp

```diff
@@ -1422,24 +1433,24 @@ IGL_INLINE bool igl::copyleft::comiso::MIQ_class<DerivedV, DerivedF, DerivedU>::
 
 template <typename DerivedV, typename DerivedF, typename DerivedU>
 IGL_INLINE void igl::copyleft::comiso::miq(
-  const Eigen::PlainObjectBase<DerivedV> &V,
-  const Eigen::PlainObjectBase<DerivedF> &F,
-  const Eigen::PlainObjectBase<DerivedV> &PD1_combed,
-  const Eigen::PlainObjectBase<DerivedV> &PD2_combed,
-  const Eigen::Matrix<int, Eigen::Dynamic, 3> &mismatch,
-  const Eigen::Matrix<int, Eigen::Dynamic, 1> &singular,
-  const Eigen::Matrix<int, Eigen::Dynamic, 3> &seams,
-  Eigen::PlainObjectBase<DerivedU> &UV,
-  Eigen::PlainObjectBase<DerivedF> &FUV,
-  double gradientSize,
-  double stiffness,
-  bool directRound,
-  int iter,
-  int localIter,
-  bool doRound,
-  bool singularityRound,
-  std::vector<int> roundVertices,
-  std::vector<std::vector<int> > hardFeatures)
+        const Eigen::PlainObjectBase<DerivedV> &V,
```

Comment:
Hi. Can you restore the original indentation here? It doesn't align to the function name so it should just be +2 spaces.